src/components/ACButton.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ACButton = (props) => {
+  const { children, clickHandler } = props;
+
+  const handleClick = () => {
+    clickHandler(children);
+  };
+
+  return (
+    <button className="modifier" onClick={handleClick}>
+      {children}
+    </button>
```

Comment:
```suggestion
    <button className="modifier" {...props} />
```
이렇게도 할 수 있을 것 같아요. (children은 명시적이게 빼는걸 선호하지만 예시를 위해서)  
jsx 문법을 바벨로 바꾸었을때 props의 타입은 무엇일까요?

Reply:
해당 부분의 코드를 babel로 변환시키니 아래와 같은 결과가 나왔습니다!

```
return /*#__PURE__*/_react.default.createElement("button", {
    className: "modifier",
    onClick: handleClick
  }, children);
```

[공식 문서](https://ko.reactjs.org/docs/react-without-jsx.html)에 의하면 JSX 엘리먼트는 React.createElement() 메서드를 호출한다고 합니다. 또한 props는 React.createElement 메서드에 객체 형식으로 전달됩니다!